### PR TITLE
test(e2e): detox tests

### PR DESCRIPTION
**Pull request by sandipndev**
This PR adds the following flows to Detox:

- [x] Login with Phone/Email Flow
- [x] Conversion Flow
- Payment Send Flow
  - [x] Intraledger using Username - BTC Amount
  - [x] Intraledger using Username - Stablesats Amount
  - [x] Intraledger using LN Invoice
  - [x] External LN Invoice
  - [x] Onchain Address
  - [ ] ~LNURL Pay~
  - [ ] ~LNURL Withdraw~
- Payment Receive Flow
  - [x] LN - BTC Amount
  - [x] LN - BTC No Amount
  - [x] LN - Stablesats Amount
  - [x] LN - Stablesats No Amount
  - [x] Onchain - BTC
  - [x] Onchain - USD
  - [ ] Paycode

https://github.com/GaloyMoney/galoy-mobile/pull/3079 should be merged first

### CODE REVIEW

File: dev/bin/dev-setup.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+BOB_USERNAME="bob_$BOB_SUFFIX"
+
+login_user "alice" "$ALICE_PHONE" "000000"
+echo "alice logged in"
+ALICE_TOKEN=$(read_value "alice")
+receive_onchain
+exec_graphql 'alice' 'user-update-username' "{\"input\": {\"username\": \"$ALICE_USERNAME\"}}"
+echo "alice set up"
+
+login_user "alice" "$BOB_PHONE" "000000"
```
Comment (UncleSamtoshi) [inline dev/bin/dev-setup.sh:24]: Should all these references be switched from "alice" to "bob"

Comment (sandipndev) [reply dev/bin/dev-setup.sh:24]: Let's modify receive_onchain on quickstart to optionally receive the token on a parameter

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -115,3 +115,4 @@ jobs:
           cd dev && tilt down || true
           docker volume prune -af || true
           docker rm -f $(docker ps -aq)
+          docker volume prune -af
```
Comment (nicolasburtey) [inline .github/workflows/e2e.yml:118]: `docker system prune -a --volumes -f`

I think this is the best comment to remove everything

**Comment by sandipndev** [comment]
Blocked on this because somehow the Galoy API in quickstart reports zero balances after funds have been sent to user's btc/usd accounts.

**Pull request closed without merging**